cmd/workspace/apps/dev.go

```diff
@@ -0,0 +1,200 @@
+package apps
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/spf13/cobra"
+)
+
+//go:embed vite-server.js
+var viteServerScript []byte
+
+// TODO: Handle multiple ports
```

Comment:
let's make this a parameter so users can run multiple dev sessions in parallel from their laptop